DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -127,7 +127,7 @@ const char* const TaggingVariableTokens[] = {
   /* [trackPtRatio]                             = */ "trackPtRatio",
   /* [trackPParRatio]                           = */ "trackPParRatio",
 
-  /* [trackIp2dVal]                             = */ "trackIp2dVal",
+  /* [trackIp2dSig]                             = */ "trackIp2dSig",
```

Comment:
I know ~nothing about the context, but want to comment about the map (as in the linked presentation) if it has not been discussed yet (if it has, sorry for the noise). The `map<const char *, T>` in slide 10 might not be what you want, as `operator[]/find` will compare the pointer, not the contents of the C-string. If you wanto to compare the C-string contents, you need to add a custom comparator to the map template parameters. And since we're talking about performance, you might also want to test sorted vector of pairs + binary search, which is often faster than similar map for searching.